sound/smooth_swing_v2.h

```diff
@@ -94,94 +111,123 @@ class SmoothSwingV2 : public SaberBasePassThrough {
           while (A.end() < 0.0) {
-            B.midpoint = A.midpoint + 180.0;
-	    Swap();
+           B.midpoint = A.midpoint + 180.0;
+           Swap();
           }
           float mixab = 0.0;
           if (A.begin() < 0.0)
-            mixab = clamp(- A.begin() / A.width, 0.0, 1.0);
-
+           mixab = clamp(- A.begin() / A.width, 0.0, 1.0);
+          
           float mixhum =
-            powf(swing_strength, smooth_swing_config.SwingSharpness);
-
+          powf(swing_strength, smooth_swing_config.SwingSharpness);
+          
           hum_volume =
-            1.0 - mixhum * smooth_swing_config.MaximumHumDucking / 100.0;
-
+          1.0 - mixhum * smooth_swing_config.MaximumHumDucking / 100.0;
+          
           mixhum *= smooth_swing_config.MaxSwingVolume;
-
+          
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
-            STDOUT.print("speed: ");
-            STDOUT.print(speed);
-            STDOUT.print(" R: ");
-            STDOUT.print(-speed * delta / 1000000.0);
-            STDOUT.print(" MP: ");
-            STDOUT.print(A.midpoint);
-            STDOUT.print(" B: ");
-            STDOUT.print(A.begin());
-            STDOUT.print(" E: ");
-            STDOUT.print(A.end());
-            STDOUT.print("  mixhum: ");
-            STDOUT.print(mixhum);
-            STDOUT.print("  mixab: ");
-            STDOUT.print(mixab);
-            STDOUT.print("  hum_volume: ");
-            STDOUT.println(hum_volume);
+           STDOUT.print("speed: ");
+           STDOUT.print(speed);
+           STDOUT.print(" R: ");
+           STDOUT.print(-speed * delta / 1000000.0);
+           STDOUT.print(" MP: ");
+           STDOUT.print(A.midpoint);
+           STDOUT.print(" B: ");
+           STDOUT.print(A.begin());
+           STDOUT.print(" E: ");
+           STDOUT.print(A.end());
+           STDOUT.print("  mixhum: ");
+           STDOUT.print(mixhum);
+           STDOUT.print("  mixab: ");
+           STDOUT.print(mixab);
+           STDOUT.print("  hum_volume: ");
+           STDOUT.println(hum_volume);
+          }
+          if (on_) {
+           // We need to stop setting the volume when off, or playback may never stop.
+           A.set_volume(mixhum * mixab);
+           B.set_volume(mixhum * (1.0 - mixab));
           }
-	  if (on_) {
-	    // We need to stop setting the volume when off, or playback may never stop.
-	    A.set_volume(mixhum * mixab);
-	    B.set_volume(mixhum * (1.0 - mixab));
-	  }
           break;
         }
         A.set_volume(0);
         B.set_volume(0);
         state_ = SwingState::OUT;
-
+        
       case SwingState::OUT:
         if (!A.isOff() || !B.isOff()) {
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
-            Serial.println("Waiting for volume = 0");
+           Serial.println("Waiting for volume = 0");
           }
         }
         PickRandomSwing();
         state_ = SwingState::OFF;
+        accent_played = false;
```

Comment:
This is set here, but never used?

Reply:
leftover from old version, should be removed.